common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -1527,65 +1527,55 @@ bool ap_wlan_hal_dwpal::read_acs_report()
 
 bool ap_wlan_hal_dwpal::read_supported_channels()
 {
-    char *reply = nullptr;
-    if (!dwpal_send_cmd("GET_HW_FEATURES", &reply)) {
```

Comment:
I think that reusing read_supported_channels is a good Idea, but i suggest that you keep the "get _hw_features" implementation and move it to a private function. 
for example `get_supported_channels_from_hostapd`

Reply:
If we decide that the new `get_radio_info` will fully replace the "current" GET_HW_FEATURES
then I agree with @tomereli 
But if there is a possibility that some other utility will need it,
as with the case of ap_wlan_hal_nl80211, where the `read_supported_channels` is used instead of the read_acs_report
Then we might still need the original code

Reply:
> But if there is a possibility that some other utility will need it,
> as with the case of ap_wlan_hal_nl80211, where the `read_supported_channels` is used

nl80211 bwl cannot be using GET_HW_FEATURES since its INTEL hostapd patch.



> I disagree - in general we don't keep dead code, we have git history for that.

then  lest remove it along with the ap_wlan_hal_nl80211 implementation and use `get_radio_info` implementation for  both BWLs

Reply:
In the new changes
both BWLs (dwpal & nl80211) use the `get_radio_info` implementation